rust/src/dns/log.rs

```diff
@@ -286,6 +287,28 @@ pub fn dns_rrtype_string(rrtype: u16) -> String {
     }.to_string()
 }
 
+fn safe_bytes_to_string(input: &[u8]) -> String {
```

Comment:
Honestly I didn't, but looking at it I still think I prefer this approach so I can see the actual "lost" bytes instead of a replacement char used for lost bytes.